apps/coscrad-frontend/src/components/commands/nack-notification.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button, Card, CardContent, Stack } from '@mui/material';
+import { IHttpErrorInfo } from '../../../../../libs/api-interfaces/src';
+
+interface NackNotificationProps {
+    _onClick: () => void;
+    errorInfo: IHttpErrorInfo;
+}
+
+export const NackNotification = ({
+    _onClick: onClick,
+    errorInfo,
+}: NackNotificationProps): JSX.Element => (
+    <Card>
```

Comment:
Also, consider using MUI Basic Dialog

Reply:
I'll add this to the story.

Reply:
[Here](https://www.pivotaltracker.com/story/show/184132185) is the story.